Source/NSDictionary+HYPImmutable.h

```diff
@@ -4,4 +4,8 @@
 
 - (NSDictionary *)hyp_dictionaryByRemovingNullItems;
 
+- (NSDictionary *)hyp_dictionaryByRemovingKey:(id <NSCopying>)key;
+- (NSDictionary *)hyp_dictionaryBySettingObject:(id)object forKey:(id <NSCopying>)key;
```

Comment:
Oh no! I see, I think it's so if you're sending a `NSMutableString` it will make an immutable copy and use that as a key.